accounts/tests.py

```diff
@@ -1,47 +1,177 @@
+from django.urls import reverse
 from django.test import TestCase
+from django.contrib.auth import get_user_model
 
+User = get_user_model()
 
 class TestSignUpView(TestCase):
     def test_success_get(self):
-        pass
+        response = self.client.get(reverse('accounts:signup'))
+        self.assertEqual(response.status_code,200)
+        self.assertTemplateUsed(response, 'accounts/signup.html')
 
     def test_success_post(self):
-        pass
+        post ={

```

Comment:
```suggestion
        post = {

```
めっちゃ細かいんですが、スペースお願いします

Reply:
修正しました。